my-trusty-scorecard/pom.xml

```diff
@@ -0,0 +1,158 @@
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.developer</groupId>
+  <artifactId>my-trusty-scorecard</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <description>Generated project my-trusty-scorecard</description>
+
+  <properties>
+    <quarkus.version>1.8.0.Final</quarkus.version>
+    <kogito.version>0.17.0</kogito.version>
+    <surefire.version>2.22.2</surefire.version>
+    <compiler.version>3.8.1</compiler.version>
+
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.release>11</maven.compiler.release> 
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>kogito-quarkus-bom</artifactId>
+        <version>${kogito.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-quarkus</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jackson</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-openapi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>tracing-decision-quarkus-addon</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-junit5</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
+    </dependency>
+<!--    &lt;!&ndash; PMML &ndash;&gt;-->
+<!--    <dependency>-->
+<!--      <groupId>org.kie.kogito</groupId>-->
+<!--      <artifactId>kogito-pmml</artifactId>-->
+<!--    </dependency>-->
+<!--    <dependency>-->
+<!--      <groupId>org.jpmml</groupId>-->
+<!--      <artifactId>pmml-model</artifactId>-->
+<!--      <version>1.5.6</version>-->
+<!--    </dependency>-->
+
+    <!-- Alternative dependencies for JPMML Evaluator, override `kie-pmml` dependency -->
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-jpmml</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jpmml</groupId>
+      <artifactId>pmml-evaluator</artifactId>
+      <version>1.5.6</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jpmml</groupId>
+      <artifactId>pmml-evaluator-extension</artifactId>
+      <version>1.5.6</version>
+    </dependency>
```

Comment:
Can you please verify if it works with `kogito-pmml` and if not what is the error?